control-plane/connect-inject/mesh_webhook_ent_test.go

```diff
@@ -496,37 +485,16 @@ func TestHandler_MutateWithNamespaces_ACLs(t *testing.T) {
 	for _, tt := range cases {
 		t.Run(tt.Name, func(t *testing.T) {
 			// Set up consul server
-			a, err := testutil.NewTestServerConfigT(t, func(c *testutil.TestServerConfig) {
+			adminToken := "123e4567-e89b-12d3-a456-426614174000"
+			testClient := test.TestServerWithConnMgrWatcher(t, func(c *testutil.TestServerConfig) {
 				c.ACL.Enabled = true
+				c.ACL.Tokens.InitialManagement = adminToken
 			})
-			a.WaitForSerfCheck(t)
-			defer a.Stop()
-
-			// Set up a client for bootstrapping
-			bootClient, err := api.NewClient(&api.Config{
-				Address: a.HTTPAddr,
-			})
-			require.NoError(t, err)
-
-			// Bootstrap the server and get the bootstrap token
-			var bootstrapResp *api.ACLToken
-			timer := &retry.Timer{Timeout: 10 * time.Second, Wait: 500 * time.Millisecond}
-			retry.RunWith(timer, t, func(r *retry.R) {
-				bootstrapResp, _, err = bootClient.ACL().Bootstrap()
-				require.NoError(r, err)
-			})
-			bootstrapToken := bootstrapResp.SecretID
-			require.NotEmpty(t, bootstrapToken)
-
-			// Set up consul client
-			client, err := api.NewClient(&api.Config{
-				Address: a.HTTPAddr,
-				Token:   bootstrapToken,
-			})
-			require.NoError(t, err)
```

Comment:
I never knew we used this "technique" in our codebase to setup a consul server with a token 😮

Reply:
I know! I was surprised too!